api-reference/beta/api/applicationsecurityprofile_list.md

```diff
@@ -0,0 +1,97 @@
+# List applicationSecurityProfiles
+
+Retrieve a list of applicationSecurityProfile objects.
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  SecurityEvents.Read.All, SecurityEvents.ReadWrite.All  |
+|Delegated (personal Microsoft account) | Not supported   |
+|Application | SecurityEvents.Read.All, SecurityEvents.ReadWrite.All |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /security/applicationSecurityProfiles
+GET /security/applicationSecurityProfiles?$top=1
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'&$top=5
+GET /security/applicationSecurityProfiles?$filter={property} eq '{property-value}'&{property} eq '{property-value}'
+```
+
+## Optional query parameters
+
+This method supports the `$count, $filter, $orderby, $select, $skip, and $top` [OData Query Parameters](http://graph.microsoft.io/docs/overview/query_parameters) to help customize the response.
+
+## Request headers
+
+| Name      |Description|
+|:----------|:----------|
+| Authorization  | Bearer {code} (required)|
+
+## Request body
+
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns a `200 OK` response code and collection of [applicationSecurityProfile](../resources/applicationsecurityprofile.md) objects in the response body.
+
+## Example
+
+### Request
+
+Here is an example of the request.
+<!-- {
+  "blockType": "request",
+  "name": "get_applicationsecurityprofiles"
+}-->
+
+```http
+GET https://graph.microsoft.com/beta/security/applicationSecurityProfiles
+```
+
+### Response
+
+Here is an example of the response. Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.

```

Comment:
Global: Change the Note into a MD-formatted note and update the text to "The response object shown here might be shortened for readability. All the properties will be returned from an actual call."